lib/scsi_defs.h

```diff
@@ -2,7 +2,7 @@
 #define LIB_SCSI_DEFS_H
 
 #include "absl/base/attributes.h"
-#include "absl/base/casts.h"
+#include "cstdint"
```

Comment:
```suggestion
#include <cstdint>
```

Always use angle brackets for system headers.